### PR TITLE
Page design

### PR DESCRIPTION
**Pull request by adejam**
# Project: NEWSWEEK CLONE
The main objective of this project is to use Bootstrap framework to build a responsive webpage by cloning [Newsweek Homepage](https://www.newsweek.com/) to demostrate the use of HTML and CSS framework in building a webpage.

The PR does the following:
- Follows the guidelines on the [Odin project assignment page](https://www.theodinproject.com/courses/html5-and-css3/lessons/using-bootstrap?ref=lnav) to build a  responsive website using Bootstrap framework by cloning [Newsweek Homepage](https://www.newsweek.com/).

- Uses CSS flexbox and float for positioning
- Uses Semantic html
- Uses CSS Media queries for responsive design
- Uses Bootstrap framework
- Uses stylint and stickler for linter 
- Uses prettier for formatting
- [Live Demo Link](https://raw.githack.com/adejam/newsweek-clone/page-design/index.html)

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+@import './main.css';
+@import './other_mq.css';
+@import 'small-screen.css' screen and (min-width: 481px);
+@import 'medium-screen.css' screen and (min-width: 768px);
+@import 'min_992.css' screen and (min-width: 992px);
+@import 'max_991.css' screen and (max-width: 991px);
+@import 'max_767.css' screen and (max-width: 767px);
+@import 'max_640.css' screen and (max-width: 640px);
+@import 'max_575.css' screen and (max-width: 575px);
+@import 'max_480.css' screen and (max-width: 480px);
```
Comment (bolah2009) [inline css/index.css]: Please refactor your CSS files

File: css/small-screen.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+.genericon_menu::before,
+.genericon_menu::after {
+  width: 30px;
+}
+
+/*
+.subscribe_block_button {
+  padding-left: 10px;
+  padding-right: 8px;
+} */
```
Comment (bolah2009) [inline css/small-screen.css]: Consider deleting commented out codes. If you need to review a previous version of your code, use git history.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,3177 @@
...
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="extra">
+        <div class="in_magazine">
+          <div>
+            <div class="magazine">
```
Comment (bolah2009) [inline index.html]: One of these should be `section` or `article` element. Let's make the overall page more semantic.


Please do this for all other occurrences

File: index.html [hunk 4]
```diff
@@ -0,0 +1,3177 @@
...
+              <div class="n_title justify-content-center align-items-center d_flex">
+                <span class="icon_n"></span>
+                in the magazine
+              </div>
```
Comment (bolah2009) [inline index.html]: This should be a heading element (h2-h6) instead, as it is the top-level element in the section (magazine).

Please do this for all other occurrences

**Review by bolah2009 — CHANGES_REQUESTED** [review]
You did a great job with the clone. The project is responsive and looks like the given design.

Please, I would recommend you refactor your CSS to not only use the mobile-first approach but also maximum of 5 breakpoints, this reduces the use if redundant CSS properties.

Also, let's use more semantic elements, please check the reviews of https://github.com/adejam/ecommerce/pull/1#pullrequestreview-434698821 and also check out [this link](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Using_HTML_sections_and_outlines)

File: css/small-screen.css (hunk 2 above)
Comment (adejam) [reply css/small-screen.css]: Thanks for the review, I'll work on this.

File: index.html (hunk 3 above)
Comment (adejam) [reply index.html]: Thanks for the review, I'll work on this.

File: index.html (hunk 4 above)
Comment (adejam) [reply index.html]: Thanks for the review, I'll work on this.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,3179 @@
...
+                  <br />
+                  Choose A Membership That's Perfect for You!
+                </p>
+                <button
+                  type="button"
+                  class="c_p subscribe_button subscribe_block_button d_ib pos_rel"
+                >
+                  subscribe
+                </button>
+              </div>
```
Comment (bolah2009) [inline index.html:3006]: There seems to be a small issue here with small screens

Steps to reproduce
- Open dev tools
- Click on `Toggle Device Toolbar`
- Select the small screen as shown in the screenshot

Expected Behaviour
 Should be responsive for small screens (~320px).


image

**Review by bolah2009 — APPROVED** [review]
LGTM 🎉 🚀 

Make changes to the issue raised before merging.

File: index.html (hunk 5 above)
Comment (adejam) [reply index.html:3006]: Thanks for the review. I'll work on this.

**Pull request merged**